### ISSUE TITLE
100% brightness triggered at startup

### ISSUE THREAD

**Issue by kupiqu** [bit-team/backintime#948]
I set regular backups every hour and it's not very comfortable when the screen goes to 100% brightness each time a new backintime instance is triggered (especially at night, when I set the screen brightness to 0%).

May it be related to the setting that disables new instances while on battery? I didn't check disabling that option though, I guess I should...

**Comment by Germar** [comment]
I don't think this has something to do with `Disable snapshots when on battery`.

I guess, this is because BiT will inhibit suspend/idle while creating a new snapshot.

**Comment by kupiqu** [comment]
Inhibit suspend/idle shouldn't per se affect brightness. I use Caffeine plus at can do it without changing current brightness status.

**Comment by emtiu** [comment]
Hi @kupiqu, does this issue still affect you? Thanks :)

**Comment by kupiqu** [comment]
> Hi @kupiqu, does this issue still affect you? Thanks :)

sorry, I am not using backintime at the moment

**Comment by emtiu** [comment]
All right, thanks for writing back anyway :) I'll keep this open, because it's almost certainly related to #1090.

**Comment by buhtz** [comment]
I am assuming the issue is fixed by #2114 .

Because reported does not use BIT anymore we can't be sure. So I tend to close the issue.

**Comment by buhtz** [comment]
Closing this ticket based on the comment above. Feel free to reopen
if the problem still exists. Thank you for your efforts. If you have
any further questions, ideas or encounter any other issues, please
don't hesitate to let us know.
